File: QuickApp/ClientApp/src/app/models/bank_branch.ts
export class bankbranch {

    constructor(Bank_Branch_ID?: number, Bank_Branch_Adress?: number, Bank_Branch_Description?: string,Bank_Name?: string,BankID?: number) {

        this.Bank_Branch_ID = Bank_Branch_ID;
        this.Bank_Branch_Adress = Bank_Branch_Adress;
        this.Bank_Branch_Description = Bank_Branch_Description;
        this.Bank_Name=Bank_Name;
        this.BankID=BankID
    }

    public Bank_Branch_ID: number;
    public BankID: number;
    public Bank_Branch_Adress: number;
    public Bank_Branch_Description: string;
    public Bank_Name: string;
   

}

File: DAL/Models/Bank Information/Bank_Branch.cs
using DAL.Models.Customer_Related;
using System;
using System.Collections.Generic;
using System.Text;

namespace DAL.Models.Bank_Information
{
    public class Bank_Branch : AuditableEntity
    {
        public int Bank_Branch_ID { get; set; }
        //foreign key from address
        public int Bank_Branch_Adress { get; set; }
        public string Bank_Branch_Description { get; set; }
        public ICollection<Bank_Account> Bank_Account { get; set; }
        public int BankID { get; set; }
        public Bank bank { get; set; }



    }
}
